pkg/segment/tracing/structs/tracestruct.go

```diff
@@ -16,47 +16,37 @@ type Status_StatusCode string
 
 const (
 	// The default status.
-	Status_STATUS_CODE_UNSET Status_StatusCode = "0"
-	Status_STATUS_CODE_OK    Status_StatusCode = "1"
-	Status_STATUS_CODE_ERROR Status_StatusCode = "2"
+	Status_STATUS_CODE_UNSET Status_StatusCode = "STATUS_CODE_UNSET"
+	Status_STATUS_CODE_OK    Status_StatusCode = "STATUS_CODE_OK"
+	Status_STATUS_CODE_ERROR Status_StatusCode = "STATUS_CODE_ERROR"
```

Comment:
Why do we want this change, and do we also need to have the 0-2 values or can we just use the `STATUS_CODE_...` values? I think the 0-2 values were added in your Search Traces PR if I'm not mistaken

Reply:
Ok, sounds good